src/styles/stars.scss

```diff
@@ -0,0 +1,30 @@
+.rate {
+  padding-left: $padding-left;
```

Comment:
The variable `$padding-left` is used but not defined. Please define this variable to avoid compilation errors.